DataConnectors/esetSmc.json

```diff
@@ -0,0 +1,117 @@
+                    "read": true,
+                    "delete": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Access to Eset SMC console",
+                "description": "Permissions to configure log export"
+            }
+        ]
+    },
+    "instructionSteps": [
+      {
+          "title": "1. Install and onboard the agent for Linux",
+          "description": "Typically, you should install the agent on a different computer from the one on which the logs are generated.\n\n>  Syslog logs are collected only from **Linux** agents.",
+          "instructions": [
+              {
+                  "parameters": {
+                      "title": "Choose where to install the agent:",
+                      "instructionSteps": [
+                          {
+                              "title": "Install agent on Azure Linux Virtual Machine",
+                              "description": "Select the machine to install the agent on and then click **Connect**.",
+                              "instructions": [
+                                  {
+                                      "parameters": {
+                                          "linkType": "InstallAgentOnLinuxVirtualMachine"
+                                      },
+                                      "type": "InstallAgent"
+                                  }
+                              ]
+                          },
+                          {
+                              "title": "Install agent on a non-Azure Linux Machine",
+                              "description": "Download the agent on the relevant machine and follow the instructions.",
+                              "instructions": [
+                                  {
+                                      "parameters": {
+                                          "linkType": "InstallAgentOnLinuxNonAzure"
+                                      },
+                                      "type": "InstallAgent"
+                                  }
+                              ]
+                          }
+                      ]
+                  },
+                  "type": "InstructionStepsGroup"
+              }
+          ]
+      },
+      {
+          "title": "2. Configure the logs to be collected",
+          "description": "Configure rsyslog to accept logs from your Eset SMC IP address.\n\n```\nsudo -i\r\n\r\n# Set ESET SMC source IP address\r\nexport ESETIP=1.2.3.4\r\n\r\n# Create rsyslog configuration file\r\ncat > \/etc\/rsyslog.d\/80-remote.conf << EOF\r\n\\$ModLoad imudp\r\n\\$UDPServerRun 514\r\n\\$ModLoad imtcp\r\n\\$InputTCPServerRun 514\r\n\\$AllowedSender TCP, 127.0.0.1, $ESETIP\r\n\\$AllowedSender UDP, 127.0.0.1, $ESETIP\r\nuser.=alert;user.=crit;user.=debug;user.=emerg;user.=err;user.=info;user.=notice;user.=warning  @127.0.0.1:25224\r\nEOF\r\n\r\n# Restart rsyslog\r\nsystemctl restart rsyslog```"
```

Comment:
Please write up all commands in the instructions on a separate line with "Copyable label" also associated with those. Refer to Syslog template as an example of how you can have a copyable label that will enable customers to copy the command and run it in their console easily without copying extra / less and run into issues.